locales/en/app.ftl

```diff
@@ -259,3 +259,71 @@ share-twitter = Most people have about 100 online accounts. Have any of yours be
 share-facebook-headline = Find out if you’ve been part of a data breach
 share-facebook-blurb = Have your online accounts been exposed in a data breach?
 og-site-description = Find out if you’ve been part of a data breach with {-product-name}. Sign up for alerts about future breaches and get tips to keep your accounts safe. 
+
+
+mozilla-security-blog = {-brand-Mozilla} Security Blog
+layout-social = Social
+show-all = Show all
+
+fxa-signup-banner-headline = Monitor threats to your online accounts.
+fxa-signup-banner-blurb = Sign up for a {-brand-fxa} to get your detailed report and alerts about new data breaches that expose your info.
+fxa-landing-blurb = 
+  Find out what hackers already know about you,
+  and learn how to stay a step ahead of them.
+
+
+fxa-scan-label = See if you’ve appeared in a data breach.
+fxa-welcome-headline = Welcome to {-product-name}.
+fxa-welcome-blurb = You’re all set to get alerts if { $userEmail } appears in a data breach.
+
+fxa-scan-another-email = Want to Check Another Email?
+fxa-scan-submit = Search {-product-name}
+
+sign-up-to-check = Sign Up to Check
+sign-in = Sign In
+
+# All letters should be capitalized.
+full-report-headline = Your {-product-name} Report
+
+# All letters should be capitalized.
+see-full-report = See Full Report
+
+#Manage Firefox Account, link to page where account holders can change their account settings.
```

Comment:
`compare-locales` on Travis complained about this line.
```
locales/en/app.ftl
    ERROR: Unparsed content "#Manage Firefox Account, link to page where account holders can change their account settings." from line 291 column 1 to line 291 column 95
```
Looks like it just needs a space after the `#` for the comment?